### ISSUE TITLE
maybe are these a bit the same mission?

### ISSUE THREAD

**Issue by hbakhtiyor** [lexborisov/Modest#28]
https://github.com/nidium/Nidium
https://github.com/litehtml/litehtml/

**Comment by lexborisov** [comment]
Hi!
No, these are not the same projects! I create web engine that fully meets the standards for rendering html like all modern browsers.
I know about the projects you listed, but nothing to do with them.

**Comment by hbakhtiyor** [comment]
hi, does modest render js? does it able to capture screenshot?

**Comment by lexborisov** [comment]
No, now I'm working on [media queries](https://drafts.csswg.org/mediaqueries/). After calculation dom and rendering (layout, events and so on), after bindings for js engine...

**Comment by vtorri** [comment]
@hbakhtiyor modest is being rewriting, look at https://github.com/lexbor/lexbor
